railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -57,6 +57,19 @@ def diff(content_path = nil)
         say credentials.content_path.read
       end
 
+      desc "fetch PATH", "Fetch a value in the decrypted credentials"
+      def fetch(path)
+        load_environment_config!
+
+        if (yaml = credentials.read)
+          YAML.load(yaml).dig(*path.split("/")).tap do |value|
```

Comment:
We should return an error if the key is missing. Also I think `.` is much more logical to nagigate than `/`.